python/paddle/fluid/executor.py

```diff
@@ -180,60 +179,24 @@ def to_name_str(var):
 
 
 class Executor(object):
-    def __init__(self, places):
-        if not isinstance(places, list) and not isinstance(places, tuple):
-            places = [places]
-
-        act_places = []
-        for each in places:
-            p = core.Place()
-            p.set_place(each)
-            act_places.append(p)
-
-        # TODO(dzhwinter) : only use the first place
-        self.executor = core.Executor(act_places[0])
-        self.places = places
+    def __init__(self, place):
+        self.place = place
+        p = core.Place()
+        p.set_place(place)
+        self.executor = core.Executor(p)
         self.program_caches = dict()
 
-    def aslodtensor(self, data):
```

Comment:
Please confirm removing parsing lod is OK?

Reply:
We plan to use the nested numpy array to represent a LodTensor, but now we use two structure to do the same thing-- an array of data, a list of lod.